svsim/src/main/scala/Backend.scala

```diff
@@ -20,18 +20,21 @@ object CommonSettingsModifications {
   */
 final class CommonSimulationSettings private[svsim] (
   val plusArgs:              Seq[PlusArg],
-  val enableWavesAtTimeZero: Boolean
+  val enableWavesAtTimeZero: Boolean,
+  val traceFileStem:         String
 ) {
```

Comment:
Given that the `traceFileStem` really has to be set, can the PR include a:

``` scala
require(traceFileStem.nonEmpty, "traceFileStem must be a non-empty String")
```

This can then promote the Backend function to not be an `Option[String]` and can just be a `String`. Given that, is the `getTraceFileStem` method even necessary anymore given that it then becomes `commonSettings.simulationSettings.traceFileStem`?